packages/ember-htmlbars/tests/helpers/input_test.js

```diff
@@ -1,492 +0,0 @@
-});
-
-QUnit.test('input placeholder is updated when setting placeholder property of view', function() {
-  equal(view.$('input').attr('placeholder'), 'Enter some text', 'renders text field with placeholder');
-});
-
-QUnit.test('input name is updated when setting name property of view', function() {
-  equal(view.$('input').attr('name'), 'some-name', 'renders text field with name');
-});
-
-QUnit.test('input maxlength is updated when setting maxlength property of view', function() {
-  equal(view.$('input').attr('maxlength'), '30', 'renders text field with maxlength');
-});
-
-QUnit.test('input size is updated when setting size property of view', function() {
-  equal(view.$('input').attr('size'), '30', 'renders text field with size');
-});
-
-QUnit.test('input tabindex is updated when setting tabindex property of view', function() {
-  equal(view.$('input').attr('tabindex'), '5', 'renders text field with the tabindex');
-});
-
-QUnit.test('specifying `on="someevent" action="foo"` triggers the action', function() {
-  expect(2);
-  runDestroy(view);
-  expectDeprecation(`Using '{{input on="focus-in" action="doFoo"}}' ('foo.hbs' @ L1:C0) is deprecated. Please use '{{input focus-in="doFoo"}}' instead.`);
-
-  controller = {
-    send(actionName, value, sender) {
-      equal(actionName, 'doFoo', 'text field sent correct action name');
-    }
-  };
-
-  view = View.create({
-    [OWNER]: owner,
-    controller,
-    template: compile('{{input type="text" on="focus-in" action="doFoo"}}', { moduleName: 'foo.hbs' })
-  });
-
-  runAppend(view);
-
-  run(function() {
-    var textField = view.$('input');
-    textField.trigger('focusin');
-  });
-});
-
-QUnit.module('{{input type=\'text\'}} - dynamic type', {
-  setup() {
-    commonSetup();
-
-    controller = {
-      someProperty: 'password'
-    };
-
-    view = View.extend({
-      [OWNER]: owner,
-      controller,
-      template: compile('{{input type=someProperty}}')
-    }).create();
-
-    runAppend(view);
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-  }
-});
-
-QUnit.test('should insert a text field into DOM', function() {
-  equal(view.$('input').attr('type'), 'password', 'a bound property can be used to determine type.');
-});
-
-QUnit.test('should change if the type changes', function() {
-  equal(view.$('input').attr('type'), 'password', 'a bound property can be used to determine type.');
-
-  run(function() {
-    set(controller, 'someProperty', 'text');
-  });
-
-  equal(view.$('input').attr('type'), 'text', 'it changes after the type changes');
-});
-
-QUnit.module('{{input}} - default type', {
-  setup() {
-    commonSetup();
-
-    controller = {};
-
-    view = View.extend({
-      [OWNER]: owner,
-      controller,
-      template: compile('{{input}}')
-    }).create();
-
-    runAppend(view);
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-  }
-});
-
-QUnit.test('should have the default type', function() {
```

Comment:
moved to `@htmlbars Helpers test: {{input}}` : [`@test default type`](https://github.com/emberjs/ember.js/pull/13194/files#diff-0caed08a992497ff104284ec6aea6d96R98)